pkg/ci/main.go

```diff
@@ -0,0 +1,72 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"log"
+	"net/http"
+	"sort"
+	"text/template"
+	"time"
+
+	"github.com/alexellis/arkade/pkg/get"
+)
+
+func URLExists(name, url, version string) error {
+	fmt.Println("--------------->>>>>>>>>>>>>>>>>>>>>>>>>", name)
+	timeout := time.Second * 10
```

Comment:
Why did you pick 10 seconds for a `HEAD`? It should be instant.

Reply:
Testing on my local machine failed with 5sec time used in rest of the code base. Will revert it back to 5 sec.